README.md

```diff
@@ -66,7 +67,7 @@ const App = React.createClass({
 });
 
 
-React.render(<App />, document.body);
+ReactDOM.render(<App />, appRoot);
```

Comment:
It's not going to 'just work' without the html as well.

For that matter, the render isn't actually key to the example. It could simply be omitted like the html is.